cvat/apps/engine/apps.py

```diff
@@ -13,3 +13,6 @@ def ready(self):
         import cvat.apps.engine.signals
         # Required in order to silent "unused-import" in pyflake
         assert cvat.apps.engine.signals
+
```

Comment:
Need to update the copyright. Please use https://marketplace.visualstudio.com/items?itemName=ymotongpoo.licenser to add/update headers automatically.